docs/lib/restapi/fragments/ios/getting-started/40_postTodo.md

```diff
@@ -0,0 +1,14 @@
+```swift
+func postTodo() {
+    let request = RESTRequest(path: "/todo", body: nil)
```

Comment:
When trying to run this im getting:
```
Fatal error: Authentication category is not configured. Call Amplify.configure() before using any methods on the category.:
```

My amplifyconfiguration.json file is not getting properly updated... hmmm

Reply:
Resolving, seems that my cli was out of date, which is was why amplifyconfiguration.json was not getting updated.